src/signal-r-response.ts

```diff
@@ -0,0 +1,4 @@
+export class SignalRResponse{
```

Comment:
Being consisten int PascalCase, please user signalr as one word. Renaming the file to signalr-response.ts and the r as lowercase in the class name.

Reply:
The tslint package is available + the npm lint command. Maybe this command can run in the PR and must be successfull before PR can be merged